tests/tck/features/schema/Schema.feature

```diff
@@ -769,6 +769,63 @@ Feature: Insert string vid of vertex and edge
       ALTER EDGE edge_not_null_default1 CHANGE (name FIXED_STRING(10) DEFAULT 10)
       """
     Then a ExecutionError should be raised at runtime: Invalid param!
+    # chinese tag and chinese prop
+    When executing query:
+      """
+      CREATE TAG 队伍(名字 string);
```

Comment:
shall we need to support use chinese characters as tag/edge/prop name directly? Is it better to use following format? 

```
CREATE TAG `队伍`(`名字` string);
```

Reply:
done